pkg/configuration/configuration.go

```diff
@@ -201,11 +201,6 @@ func New(configFilePath string, cl client.Client) (*Config, error) {
 	return c, nil
 }
 
-func getRegistrationEnvVarKey(key string) string {
```

Comment:
good catch

Reply:
thanks, but credit goes to `golangci-lint` 😂